docker-compose.yml

```diff
@@ -52,27 +64,27 @@ services:
     depends_on:
       - postgres
 
-  worker_high:
-    <<: *robotoff-base
-    deploy:
-      mode: replicated
-      replicas: 2
-    command: python -m robotoff run-worker robotoff-high
-    environment: *robotoff-base-env
-    depends_on:
-      - postgres
-    mem_limit: 8g
+  worker_high_1:
+    <<: *robotoff-worker
+    container_name: ${COMPOSE_PROJECT_NAME:-robotoff}_worker_high_1
+    # Each worker (whether it's a high or low priority worker) listens to a
+    # single high priority queue
```

Comment:
I would put that comment below on  _low_1 where it's more easy to miss the fact it's listening to a high queue.